recipes/libspatialite/all/conanfile.py

```diff
@@ -85,28 +85,30 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("sqlite3/3.42.0")
-        self.requires("zlib/1.2.13")
+        # Included in public spatialite/sqlite.h
+        # https://www.gaia-gis.it/fossil/libspatialite/file?name=src/headers/spatialite/sqlite.h&ci=tip
+        self.requires("sqlite3/3.44.2", transitive_headers=True, transitive_libs=True)
+        self.requires("zlib/[>=1.2.11 <2]")
         if self.options.with_proj:
-            self.requires("proj/9.1.1")
+            self.requires("proj/9.3.1")
         if self.options.with_iconv:
             self.requires("libiconv/1.17")
         if self.options.with_freexl:
-            self.requires("freexl/1.0.6")
+            self.requires("freexl/2.0.0")
         if self.options.with_geos:
-            self.requires("geos/3.11.1")
+            self.requires("geos/3.12.0")
         if self.options.get_safe("with_rttopo"):
             self.requires("librttopo/1.1.0")
         if self.options.with_libxml2:
-            self.requires("libxml2/2.10.3")
+            self.requires("libxml2/2.12.4")
```

Comment:
GDAL uses libxml2/2.12.3. It will require another PR to align it. All other dependencies are aligned with GDAL requirements. Nice job.